[feature] print multiple docs from multiple doctypes

codingCoffee opened the pull request:
Please read the [Pull Request Checklist](https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist) to ensure you have everything that is needed to get your contribution merged.

codingCoffee committed:
[feature] print multiple docs from multiple doctypes

rmehta commented on frappe/utils/print_format.py:
```diff
@@ -20,14 +20,21 @@ def download_multi_pdf(doctype, name, format=None):
+		for doctype_name in doctype:
+			for doc_name in doctype[doctype_name]:
+				output = frappe.get_print(doctype_name, doc_name, format, as_pdf = True, output = output)
+		frappe.local.response.filename = "{}.pdf".format(frappe.session.user.replace('@', '-'))
```
why user name as file name?

Maybe explicitly ask filename

Also update the docstring

codingCoffee committed:
added docstring to download multi pdf

- also made the PDF name as a parameter which is passed by the user

codingCoffee commented:
@rmehta made the necessary changes

codingCoffee committed:
multi fixes

- [fix] removed _ as a variable since it is used in translations
- linting fixes
- added error logging for permission errors
